ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -29,22 +29,21 @@
         "id": "title"
```

Comment:
<p>Shall we fix this links in the notebook while we're at it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think the updated links are correct? They're only showing up weird in ReviewNB. They look OK here: https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/update-bq-notebook/ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb</p>